Allow sorting Identification lists by confidence score

philomates opened the pull request:
Adding back code that was removed when things were reorganized a while back:
- parcel tests
- equality implementations, which are needed for tests
- comparator code for sorting identification lists

Also fixed a getter method name typo in https://github.com/SimPrints/LibSimprints/pull/1/commits/4b299ff5e9c5f89d1b39b702db08150abdfe84e0

JuliaKraus commented:
Why am I still getting these e-mails?

:-(


Julia Kraus
*www.simprints.com <http://www.simprints.com/>*  *@simprints
<https://twitter.com/Simprints>*
*Facebook <https://www.facebook.com/simprints/>*Learn more in *The
<http://www.economist.com/news/britain/21661670-political-philosophy-britains-most-successful-city-getting-cambridge>
**Economist
<http://www.economist.com/news/britain/21661670-political-philosophy-britains-most-successful-city-getting-cambridge>*
, *Bloomberg
<http://www.bloomberg.com/news/videos/2014-11-13/how-this-40-finger-scanner-could-transform-health-care>*,
and *BBC <http://www.bbc.co.uk/programmes/p02lnjxk>*

On 8 March 2016 at 11:27, Tristram Norman <notifications@github.com> wrote:

> Merged #1 <https://github.com/SimPrints/libSimprints/pull/1>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SimPrints/libSimprints/pull/1#event-581594439>.
>

AlanDeanKemp committed:
Create and maintain as separate library

AlanDeanKemp committed:
Remove log messages

AlanDeanKemp committed:
Changed confidence from float to double

AlanDeanKemp committed:
Remove app directory

AlanDeanKemp committed:
Remove legacy code

AlanDeanKemp committed:
Remove legacy code

AlanDeanKemp committed:
Remove legacy code, update .gitignore

AlanDeanKemp committed:
Changed confidence from float to double

AlanDeanKemp committed:
Changed finger id constants to match (preferred) order of scanning

AlanDeanKemp committed:
Changed type of confidence to float (for now)

AlanDeanKemp committed:
Fixed typo in identification

AlanDeanKemp committed:
Libsimprints reduced to identification and registration classes only

AlanDeanKemp committed:
Changed typo for identification intent

philomates committed:
Add back old functionality

 - parcel tests
 - equality implementations, which are needed for tests
 - comparator code for sorting identification lists

philomates commented on libsimprints/src/main/AndroidManifest.xml:
```diff
@@ -1,12 +1 @@
-        android:supportsRtl="true"
-        >
-
-    </application>
```
As far as I know, if `application` is present this can't be imported as a library into another app.

philomates commented on libsimprints/src/main/res/values/strings.xml:
```diff
@@ -1,3 +0,0 @@
-<resources>
-    <string name="app_name">LibSimprints</string>
-</resources>
```
Unused after `application` is removed from `AndroidManifest.xml`

philomates commented on libsimprints/src/main/java/com/simprints/libsimprints/Identification.java:
```diff
@@ -69,4 +69,34 @@ public float getConfidence() {
+        } else {
+            return -1;
+        }
+    }
```
Sorting identification results is used by CommCare to show top results first

philomates committed:
Fix getter method name typo

philomates committed:
Update library versions
